hera_cal/redcal.py

```diff
@@ -1454,11 +1455,11 @@ def redundantly_calibrate(data, reds, freqs=None, times_by_bl=None, fc_conv_crit
         times_by_bl = data.times_by_bl
 
     # perform firstcal
-    rv['g_firstcal'] = rc.firstcal(data, freqs, maxiter=fc_maxiter, conv_crit=fc_conv_crit)
+    rv['fc_meta'], rv['g_firstcal'] = rc.firstcal(data, freqs, maxiter=fc_maxiter, conv_crit=fc_conv_crit)
     rv['gf_firstcal'] = {ant: np.zeros_like(g, dtype=bool) for ant, g in rv['g_firstcal'].items()}
 
     # perform logcal and omnical
-    log_sol = rc.logcal(data, sol0=rv['g_firstcal'])
+    log_sol = _, rc.logcal(data, sol0=rv['g_firstcal'])
```

Comment:
I'm not sure I understand what is "_" for?

Reply:
It's just a convention for a variable that is returned but not used later.